website/docs/validating-admission-policy.md

```diff
@@ -112,24 +112,26 @@ spec:
         rego: |
           ...
 ```
-With this new engine and source added to the constraint template, now Gatekeeper webhook, audit, and shift-left can validate resources with these new CEL-based rules. 
+With this new engine and source added to the constraint template, now Gatekeeper webhook, audit, and shift-left can validate resources with these new CEL-based rules.
 
 ## Policy updates to generate Validating Admission Policy resources
 
-For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook. By default, Gatekeeper is configured to generate K8s Validating Admission Policy resources if the `gatekeeper.sh/use-vap` label is used. In the event K8s Validating Admission Controller fails open, then Gatekeeper admission webhook can act as a backup. Default value for this feature flag is `--vap-enforcement=GATEKEEPER_DEFAULT`. 
+For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook. By default, Gatekeeper is configured to generate K8s Validating Admission Policy resources if `generateVAP: true` is set on `ConstraintTemplate`. In the event K8s Validating Admission Controller fails open, then Gatekeeper admission webhook can act as a backup. Default value for this feature flag is `--default-create-vap-for-templates=false`.
```

Comment:
For some policies, you may want admission requests to be handled by the K8s Validating Admission Controller instead of the Gatekeeper admission webhook. 

Gatekeeper is configured to generate K8s Validating Admission Policy (VAP) resources for all constraint templates globally if `--default-create-vap-for-templates=true` flag is set. This flag defaults to `false` at this time to not generate VAP resources by default.

If you would like to override this flag's behavior for any constraint templates, you can set `generateVAP` explicitly on per constraint template level.

 ```yaml
 spec:
   targets:
     - target: admission.k8s.gatekeeper.sh
       code:
         - engine: K8sNativeValidation
           source:
             generateVAP: true
             ...
 ```

Constraints will follow the behavior defined by `--create-vap-binding-for-constraints` flag to generate K8s Validating Admission Policy Binding. By default, `--create-vap-binding-for-constraints` is set to `false`.

:::tip
 In the event K8s Validating Admission Controller fails open, then Gatekeeper admission webhook can act as a backup.
 :::

Reply:
i think we can remove the section after this to simplify it a bit.

Reply:
Thanks for the suggestion. Updated the docs.